.config/pydoclint-baseline.txt

```diff
@@ -0,0 +1,3 @@
+tests/conftest.py
+    DOC503: Method `Infrastructure.__post_init__` exceptions in the "Raises" section in the docstring do not match those in the function body Raises values in the docstring: ['ValueError', 'ValueError', 'ValueError']. Raised exceptions in the body: ['ValueError'].
```

Comment:
Bug open with them, seems more like a false positive which does not work with noqa.

Reply:
got it, josh is good people.